src/utils/dictionaries/trading-pair-dictionary.ts

```diff
@@ -11,14 +10,12 @@ export enum TradingPairType {
   BPRO_XUSD = 'BPRO_XUSD',
   BPRO_USDT = 'BPRO_USDT',
   BPRO_DOC = 'BPRO_DOC',
+  SOV_XUSD = 'SOV_XUSD',
+  SOV_DOC = 'SOV_DOC',
+  BPRO_SOV = 'BPRO_SOV',
```

Comment:
This is just a question but why do we have `SOV_XUSD`, `SOV_DOC` and then suddenly `BPRO_SOV`? Or the order doesn't matter?

Reply:
Yep, order doesn't matter but will reorder it because of readability

Reply:
ok thanks :)